src/components/RoarDataTable.vue

```diff
@@ -249,8 +249,8 @@ import _find from 'lodash/find';
 import _filter from 'lodash/filter';
 import _toUpper from 'lodash/toUpper';
 import _startCase from 'lodash/startCase';
-import _lowerCase from 'lodash/lowerCase';
-import { taskFilterBlacklist } from '@/helpers/reports';
+// import _lowerCase from 'lodash/lowerCase';
```

Comment:
Maybe we can keep this in here to handle the fallback case in the switch statement below
```suggestion
import _lowerCase from 'lodash/lowerCase';
```